### DISCUSSION TITLE
Error when updating Crazyradio firmware

### DISCUSSION THREAD

**Discussion by GideonLombardo** [USC-ACTLab/crazyswarm#595] [Q&A] [answered]
Hello, I am trying to setup Crazyswarm on my machine. When I go to configure the firmware for the Crazyradio, I type the command:
sudo python3 crazyradio-firmware/usbtools.nrfbootload.py flash prebuilt/cradio.bin

I get the following error message:
Found nRF24LU1 bootloader version 18.0
Flashing:
Traceback (most recent call last):
  File "crazyradio-firmware/usbtools/nrfbootload.py", line 130, in <module>
    f = open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'prebuilt/cradio.bin'

I know the the prebuilt/cradio.bin path exists in my crazyswarm directory. Is there something I need to edit in the nrfbootload.py file?

**Comment by whoenig** [comment, marked as answer]
There should be no edits needed. From what I remember, there you used to be compatibility issues with python2/3. Try specifying the full path (and not a relative path).

**Reply by GideonLombardo** [reply]
This worked. Thank you very much.
